pusher/src/Services/AdminWokaService.ts

```diff
@@ -0,0 +1,64 @@
+import axios from "axios";
+import { ADMIN_API_TOKEN, ADMIN_API_URL } from "../Enum/EnvironmentVariable";
+import { WokaDetailsResult, WokaList } from "../Enum/PlayerTextures";
+import { WokaServiceInterface } from "./WokaServiceInterface";
+
+class AdminWokaService implements WokaServiceInterface {
+    /**
+     * Returns the list of all available Wokas for the current user.
+     */
+    getWokaList(token: string): Promise<WokaList | undefined> {
+        return axios
+            .get(`${ADMIN_API_URL}/api/woka-list/${token}`, {
+                headers: { Authorization: `${ADMIN_API_TOKEN}` },
+            })
+            .then((res) => {
+                return res.data;
```

Comment:
We need to add a type guard here (we have no guarantee "res.data" will indeed be a `WokaList`